### PR TITLE
Update Italian translations

### PR DESCRIPTION
**Pull request by DomeT99**
Hi, I closed the other pull request and reopened this one with fixes.

### CODE REVIEW

**Comment by mia-pi-git** [comment]
You don't need to make a new PR for fixes - just push to the old one in the future.

File: translations/italian/helptickets.ts [hunk 1]
```diff
@@ -147,9 +147,9 @@ export const translations: Translations = {
 		"${targetUser ? targetUser.name : target} is not ticket banned.": "${targetUser ? targetUser.name : target} non è bannato dall'aprire ticket.",
 		"${targetUser ? targetUser.name : target}'s ticket ban is already expired.": "Il ticketban di ${targetUser ? targetUser.name : target} è già terminato.",
 		"You are already ignoring help ticket notifications. Use /helpticket unignore to receive notifications again.": "Stai già ignorando le notifiche dei ticket help. Usa /helpticket unignore per ricevere nuovamente le notifiche.",
-		"You are now ignoring help ticket notifications.": "Stai ora ignorando le notifiche dei ticket help.",
+		"You are now ignoring help ticket notifications.": "Stai ora ignorando le notifiche dei ticket di aiuto.",
```
Comment (f0ffee) [inline translations/italian/helptickets.ts:150]: This was originally translated by a person who actually handled help tickets, and he left this in English on purpose.

File: translations/italian/helptickets.ts [hunk 2]
```diff
@@ -147,9 +147,9 @@ export const translations: Translations = {
...
 		"You are not ignoring help ticket notifications. Use /helpticket ignore to stop receiving notifications.": "Non stai ignorando le notifiche dei ticket help. Usa /helpticket ignore per non ricevere più le notifiche.",
-		"You will now receive help ticket notifications.": "Non riceverai più alcuna notifica dei ticket help.",
+		"You will now receive help ticket notifications.": "Non riceverai più alcuna notifica dei ticket di aiuto.",
```
Comment (f0ffee) [inline translations/italian/helptickets.ts:152]: This was originally translated by a person who actually handled help tickets, and he left this in English on purpose.

File: translations/italian/main.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ export const translations: Translations = {
 	name: "Italian",
 	strings: {
 		"namelocked": "namelockato",
-		"locked": "lockato",
+		"locked": "bloccato",
```
Comment (f0ffee) [inline translations/italian/main.ts:7]: The commonly used term for this is "lockato" in Italian

File: translations/italian/main.ts [hunk 4]
```diff
@@ -16,13 +16,13 @@ export const translations: Translations = {
 
 		"<strong>Global ranks</strong>": "<strong>Ruoli Globali</strong>",
 		"+ <strong>Global Voice</strong> - They can use ! commands like !groups": "+ <strong>Global Voice</strong> - Possono utilizzare i comandi con !, come !groups",
-		"% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts": "% <strong>Global Driver</strong> - Come i Global Voice, e possono lockare gli utenti e controllare i loro alts",
+		"% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts": "% <strong>Global Driver</strong> - Come i Global Voice, e possono bloccare gli utenti e controllare i loro alts",
```
Comment (f0ffee) [inline translations/italian/main.ts:19]: The commonly used term for this is "lockare" in Italian

File: translations/italian/main.ts [hunk 5]
```diff
@@ -34,7 +34,7 @@ export const translations: Translations = {
 		"For details of a specific command, use something like: /help data": "Per i dettagli di un comando specifico, usa ad esempio /help data",
 
 		"COMMANDS": "COMANDI",
-		"BATTLE ROOM COMMANDS": "COMANDI PER LA CHAT IN BATTLE",
+		"BATTLE ROOM COMMANDS": "COMANDI PER LA CHAT IN BATTAGLIA",
```
Comment (f0ffee) [inline translations/italian/main.ts:37]: We commonly refer to this as "battle" even in Italian

File: translations/italian/main.ts [hunk 6]
```diff
@@ -61,7 +61,7 @@ export const translations: Translations = {
 		"- /warn OR /k <em>username</em>: warn a user and show the Pok&eacute;mon Showdown rules": "- /warn O /k <em>nome utente</em>: d&agrave; un avviso all'utente in questione e gli mostra le regole di Pok&eacute;mon Showdown",
 		"- /mute OR /m <em>username</em>: 7 minute mute": "- /mute O /m <em>nome utente</em>: muta per 7 minuti",
 		"- /hourmute OR /hm <em>username</em>: 60 minute mute": "- /hourmute O /hm <em>nome utente</em>: muta per 60 minuti",
-		"- /unmute <em>username</em>: unmute": "- /unmute <em>nome utente</em>: rimuove il mute",
+		"- /unmute <em>username</em>: unmute": "- /unmute <em>nome utente</em>: rimuove il muto",
```
Comment (f0ffee) [inline translations/italian/main.ts:64]: We commonly refer to this as "mute" even in Italian

File: translations/italian/main.ts [hunk 7]
```diff
@@ -75,8 +75,8 @@ export const translations: Translations = {
 		"- /staffintro <em>intro</em>: set the staff introduction that will be displayed for all staff joining the room": "- /staffintro <em>intro</em>: imposta l'introduzione dello staff, che verr&agrave; mostrata a tutti i componenti dello staff una volta che entrano in room",
 		"- /roomsettings: change a variety of room settings, namely modchat": "- /roomsettings: cambia una variet&agrave; di impostazioni della room, pi&ugrave; comunemente la modchat",
 
-		"<strong>Room owners (#)</strong> can also use:": "<strong>Gli owner della room (#)</strong> possono inoltre usare:",
-		"- /roomintro <em>intro</em>: set the room introduction that will be displayed for all users joining the room": "- /roomintro <em>intro</em>: imposta l'introduzione della room che verr&agrave; mostrata a tutti gli utenti quando entrano in room",
+		"<strong>Room owners (#)</strong> can also use:": "<strong>I proprietari della room (#)</strong> possono inoltre usare:",
```
Comment (f0ffee) [inline translations/italian/main.ts:78]: We commonly refer to them as "owner" in Italian

**Comment by DaWoblefet** [comment]
The vast majority of these translations seem to be unnecessary, as f0fee says. I'll draft this for now until comments are addressed.

**Pull request closed without merging**